### PR TITLE
[FIX #3651] Change console icon

### LINKED ISSUE #3651: Make Console icon style look like dev tool
### User Story

As a developer, I want to see that console is more look like terminal/bash so that  I can trust it more and could find it through a list of chats easily.

### Description

*Type*: Feature

I would like to restyle a Console user pick and replaced empty state text to achieve the user story. 

The scope of the issue:
—  Rename Console to Status Console
—  Delete small icon with two dots 
—  Change the userpick and double check that size of it is 40px x 40 px
—  Change empty screen text to `Look under the hood! Console is a javascript runtime environment that exposes the whole web3 API. <br>Type "web3." to get started.`
—  Double check that console userpick is replaced in the chat view

#### Actual behavior
artboard 2

#### Expected behavior 
artboard 2 copy

### Additional Information
 Operating System: Android|iOS

### PR DESCRIPTION
**Pull request by msuess**
fixes #3651

### Summary:

Change the Console chat icon to look more like a dev tool.

### Todo:
- [x] change console icon
- [x] remove dapp badge from console icon
- [x] change console name to "Status Console"
- [x] change empty screen text in console chat

### Steps to test:
- Open Status
- Navigate to home screen
- Make sure the icon looks as described in #3651 
- Make sure that the contact name is "Status Console"
- Open the chat with no messages
- Make sure the empty screen text reads `Look under the hood! Console is a javascript runtime environment that exposes the whole web3 API. <br>Type "web3." to get started.` as described in #3651

status: ready

### CODE REVIEW

**Comment by msuess** [comment]
@denis-sharypin do you have the new pic in 41, 82 and 123px side length for the imageset?

**Comment by msuess** [comment]
Should the name "Status Console" be translatable into other languages?

**Comment by msuess** [comment]
Hmm, I just realized that it's probably enough to set `:dapp?` to `false` in `console.cljs`, instead of hiding the badge dynamically. Will fix this tomorrow. Signing off for today.

EDIT: nope, this breaks all the "special" commands such as `/faucet`

**Comment by denis-sharypin** [comment]
@msuess  sure!
console
console 2x
console 3x

**Comment by msuess** [comment]
@denis-sharypin umm, I just realized that I also need it in 164 and 62 😰 Sorry for the delay. Thanks!

**Comment by denis-sharypin** [comment]
@msuess np
group 1 5x
group 4x

**Comment by msuess** [comment]
@denis-sharypin I'm done with this, do you mind taking a look if everything is as you expect?

**Comment by denis-sharypin** [comment]
@asemiankevich could you kindly check this PR out?

File: src/status_im/chat/screen.cljs [hunk 1]
```diff
@@ -2,6 +2,7 @@
   (:require-macros [status-im.utils.views :refer [defview letsubs]])
   (:require [clojure.string :as string]
             [re-frame.core :as re-frame]
+            [status-im.constants :as const]
```
Comment (jeluard) [inline src/status_im/chat/screen.cljs]: Can you use `constants` as alias?

**Comment by flexsurfer** [comment]
amazing work @msuess , thanks, we are trying to get rid of "native" images, could you please try to remove "native" images and use image from `resources` same as other dapps `"photo-path": "contacts://console",`? thanks

**Comment by asemiankevich** [comment]
@denis-sharypin i will check it when it is ready! :)

**Comment by msuess** [comment]
@denis-sharypin could I have the console icon one more time in 192? thanks, and sorry for the inconvenience!

**Comment by msuess** [comment]
@flexsurfer do you mind elaborating on this? I've changed the `:photo-path` to `contacts://console` and put the png into `resources/contacts/console.png`, but it's not showing up in the UI.

File: src/status_im/chat/screen.cljs (hunk 1 above)
Comment (msuess) [reply src/status_im/chat/screen.cljs]: Done!

**Comment by msuess** [comment]
@flexsurfer thanks for the feedback, consider it done!

**Comment by flexsurfer** [comment]
great job @msuess ,could you please resolve conflicts

**Comment by denis-sharypin** [comment]
@msuess Sorry I haven't prepared everything in advance

**Comment by msuess** [comment]
@flexsurfer since I removed the native image and used the one from `resources`, the icon does not show up anymore in the chat (only in the chat list). do you have any idea what this could be about?

EDIT:
There seems to be some logic in `chat-icon/screen.cljs` that handles loading the pic from resources:
```clojure
(if (string/starts-with? photo-path "contacts://")
                  (->> (string/replace photo-path #"contacts://" "")
                       (keyword)
                       (get resources/contacts))
                  {:uri photo-path})
```
This does not seem to be reflected in `views/message/message.cljs`. Should I add something like this there as well? How do you handle this for other pre-defined contacts, eg. `jarrad`?

**Comment by msuess** [comment]
@flexsurfer could you kindly provide some guidance here? Thanks!

**Comment by flexsurfer** [comment]
@msuess try to create new account and check , it should work

**Comment by msuess** [comment]
@flexsurfer that's what I did, it's not working :(

**Comment by flexsurfer** [comment]
@msuess looks like it were broken some time ago, so yes we need to fix this and add this logic to messages, thanks

**Comment by msuess** [comment]
@flexsurfer I added the logic [here](https://github.com/status-im/status-react/pull/3671/files#diff-714969497f614cba3dfc82123b601492R248), but it's not very DRY since it repeats some of the code in [components/chat_icon/screen.cljs](https://github.com/status-im/status-react/blob/38c9bb7266a05778a61e3e7fbb41d121a7831209/src/status_im/ui/components/chat_icon/screen.cljs#L17). It does not do the exact same thing though as it's not using the identicon iff there is no photo path.

Also, there is some similar code in [message-delivery-status](https://github.com/status-im/status-react/blob/38c9bb7266a05778a61e3e7fbb41d121a7831209/src/status_im/chat/views/message/message.cljs#L223), should we use the same logic here?

I'm a little bit afraid to DRY this up as I can't properly test this since I'm somehow unable to add my account that I'm using in the app from the play store on my phone. I'm guessing this is because those are not the same versions? Or is this a bug? Thanks!

**Comment by flexsurfer** [comment]
that's fine @msuess we plan to rework this /message/message.cljs later anyway, so it's fine to have same code, and we don't need this in message-delivery-status, so we can move this to test, could you please elaborate about an account problem, why are you trying to add account from play store, and how are you trying to do this, thanks

**Comment by msuess** [comment]
@flexsurfer sorry if I wasn't clear enough.

I installed Status from the Play Store on my Android phone and created an account (A) from within the app. I also have the development version of Status running on an Android emulator on my Laptop with another account (B). If I enter the contact code from B on my phone it adds a new private chat on my phone, but I am unable to receive messages that I sent from my phone within the emulator. The same goes the other way around.

I hope I'm being clear this time, let me know if you have questions.

**Comment by msuess** [comment]
@flexsurfer @denis-sharypin I'm done with my changes :)

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
@msuess great job, about accounts, we have different version of whisper in play store and in development, that's why you can't messaging

**Comment by asemiankevich** [comment]
@msuess could you please add `<br>` tag before `Type` so the whole phrase `Type "web3. to get started" will be in new line? 
image

**Comment by msuess** [comment]
@asemiankevich I left it out because manual line breaks have led to broken formatting in the past with some screen sizes (see 2nd screenshot in #3629).

Do you want me to add it anyway?

**Comment by asemiankevich** [comment]
@msuess i see, no, lets leave it as is. Thank you!

**Pull request merged**